README.md

```diff
@@ -87,15 +109,20 @@ $ git clone https://github.com/paritytech/parity-ethereum
 $ cd parity-ethereum
 
 # build in release mode
-$ cargo build --release --features final
+$ cargo build --release -p features final
 ```
+For tests, also update submodules:
+
+```$ git submodule init```
+```$ git submodule update```
 
 This produces an executable in the `./target/release` subdirectory.
 
 Note: if cargo fails to parse manifest try:
 
 ```bash
 $ ~/.cargo/bin/cargo build --release
+$ ./target/release/features final --help
```

Comment:
this is also wrong the `binary` will be named `parity`.

there's no CLI option `final` is just a feature flag AFAIK